src/Illuminate/Support/Facades/Crypt.php

```diff
@@ -6,7 +6,7 @@
  * @method static string encrypt(string $value, bool $serialize = true)
  * @method static string decrypt(string $payload, bool $unserialize = true)
  *
```

Comment:
I believe Facade is supposed to hint the available methods on the interface and the new ones (generateKey, isValid) aren't present here.